maintainers/maintainer-list.nix

```diff
@@ -6599,6 +6599,16 @@
     githubId = 1387206;
     name = "Mike Sperber";
   };
+  mikroskeem = {
+    email = "mikroskeem@mikroskeem.eu";
+    github = "mikroskeem";
+    githubId = 3490861;
+    name = "Mark Vainomaa";
+    keys = [{
+      longkeyid = "rsa4096/0xDA015B05B5A11B22";
+      fingerprint = "DB43 2895 CF68 F0CE D4B7  EF60 DA01 5B05 B5A1 1B22";
+    }];
+  };
```

Comment:
Will this not clash with the other PR https://github.com/NixOS/nixpkgs/pull/124248?

Reply:
It will. If one of these two get merged, I'll rebase.

Reply:
Or well, probably git is smart enough to figure out that commits are 1:1 (I just cherry picked the commit)

Reply:
it should be able to filter it out

Reply:
the rebase merge only committed 1 commit